tests/FakeItEasy.Tests.Approval.netstd/ApiApproval.cs

```diff
@@ -0,0 +1,33 @@
+namespace FakeItEasy.Tests.Approval
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using System.Runtime.CompilerServices;
+    using ApprovalTests;
+    using ApprovalTests.Reporters;
+    using PublicApiGenerator;
+    using Xunit;
+
+    public class ApiApproval
+    {
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveApiNetStd()
+        {
+            // Approvals and PublicApiGenerator aren't available for .NET Core, so
+            // we'll load the .NET Standard FakeItEasy assembly from disk and
+            // examine it.

```

Comment:
Do you mean that it works with an assembly loaded with `Assembly.LoadFrom`, but not with an assembly already loaded in the app domain?